mycity/mycity/intents/get_alerts_intent.py

```diff
@@ -149,12 +191,32 @@ def _create_response_object() -> MyCityResponseDataModel:
     return mycity_response
 
 
+def list_alerts_output(alerts: typing.Dict) -> typing.AnyStr:
+    """
+    Output list of alerts to pick for when there's multiple alerts.
+
+    :param alerts: pruned alert dictionary
+    :return: a string containing all alerts to decide on.
+    """
+    logger.debug('alerts: ' + str(alerts))
+    alert_count = len(alerts)
+    alert_list = ""
+    for i, alert in enumerate(alerts.keys()):
+        if i + 1 == alert_count:
+            alert_list += 'and '
+        alert_list += alert.capitalize()
+        if i + 1 < alert_count:
+            alert_list += ', '
```

Comment:
I think there is a more python way of doing this using [string's join function](https://docs.python.org/2/library/stdtypes.html#str.join). Didn't test this, but something like:
`alert_list = ', '.join(alerts.keys()[:-1]) + "and " + alerts.keys()[-1]`